lib/tasks/make_background.rake

```diff
@@ -0,0 +1,40 @@
+task make_background: :environment do
```

Comment:
what does  ":environment" do here?

Reply:
This gives the task access to the models, like `Background` and `Sample`